### PR TITLE
Press release widget link

### PR DESCRIPTION
**Pull request by misfist**
Changes to address bug report:

> There's a problem with press mentions on report pages (see In the Press here: https://public-accountability.org/report/the-anti-green-new-deal-coalition/ ) and on this page: 
> https://public-accountability.org/category/press-mention/
> They link to a page on PAI for that press mention, rather than sending you to the external press link. Can you fix?

### CODE REVIEW

File: includes/extras.php [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+ * Filter Link for Press Mentions
+ * 
+ * @link https://developer.wordpress.org/reference/hooks/post_link/
+ *
+ * @param string $url
+ * @param obj $post
+ * @param boolean $leavename
+ * @return string $url
+ */
+function pai_post_link_press( $url, $post ) {
```
Comment (misfist) [inline includes/extras.php]: This function modifies the URL for any press mention post if it has `source_url`.

**Comment by aepyornis** [comment]
hey @misfist   thanks!!!

are you able to resolve the conflicts in style.css?

**Comment by misfist** [comment]
@aepyornis - Er... The only thing that changed is the version number...

Should  be ` Version:      0.2.2`

**Comment by misfist** [comment]
@aepyornis - OK. Fixed.

**Comment by aepyornis** [comment]
amazing, thank you!

**Pull request merged**